vue-custom-bootstrap/src/components/CheckboxInput.vue

```diff
@@ -4,16 +4,20 @@
     :invalid-feedback="errorText"
     :state="hasError()"
     :label-for="id"
-    :label="label"
     class="mb-4"
   >
```

Comment:
This block could be:

```
 <label :for="id" class="font-weight-bold">{{label}}</label>
```

If the label doesn't wrap the input and doesn't have the "for" attribute, it won't be associated with the input, which breaks accessibility.

Then the "font-weight-bold" is a reusable CSS class from bootstrap so we don't have write our own: https://getbootstrap.com/docs/4.3/utilities/text/

Reply:
The idea was to have one class for the question label that can be modified in one place only: app.scss

Reply:
Makes sense, I'm good with either way for now. We can talk to Sabrina later on and see what all the different typography is, so we can organize it better.